Lib/test/test_re.py

```diff
@@ -1227,7 +1227,7 @@ def test_pickling(self):
             pickled = pickle.dumps(oldpat, proto)
             newpat = pickle.loads(pickled)
             self.assertEqual(newpat, oldpat)
-        # current pickle expects the _compile() reconstructor in re module
+        # previous pickles may expect the _compile() reconstructor in re module
```

Comment:
Is it worth adding a test for pickles that expect ``_compile()``? ("No" is a valid answer)

Reply:
I've added it in https://github.com/python/cpython/pull/122357/commits/df79dd5cc1467f4cfc9e9902d216498e7391f51e